core/src/main/java/tachyon/master/MasterInfo.java

```diff
@@ -1376,6 +1379,42 @@ public int getFileId(TachyonURI path) throws InvalidPathException {
     }
     return ret;
   }
+  
+  /**
+   * @return the total bytes on each storage tier.
+   */
+  public List<Long> getTotalBytesOnTiers() {
+    List<Long> ret = new ArrayList<Long>();
```

Comment:
Could the Arraylist length be determined when we instantiate it (# of storage level aliases)? Also, does the array list need to be defaulted to zero from index 0 to # of storage level aliases everywhere (as opposed to just the initial arraylist of the worker)?